package.json

```diff
@@ -22,6 +22,7 @@
     "chai": "^3.0.0",
     "coffee-script": "~1.10.0",
     "mocha": "^3.0.2",
-    "resin-lint": "^1.2.2"
+    "resin-lint": "^1.2.2",
+    "versionist": "^2.11.0"
```

Comment:
I'm a bit confused, is this to add VersionBot or is this to actually use `versionist`? If it's to just add VersionBot, as everything's already enabled and the `package.json`, `CHANGELOG.md` and tags are up to date, it shouldn't need a PR at all.